lib/matplotlib/tests/test_image.py

```diff
@@ -1118,3 +1118,9 @@ def test_exact_vmin():
 
     # check than the RBGA values are the same
     assert np.all(from_image == direct_computation)
+
+
+@pytest.mark.network
```

Comment:
Gives a warning in CI:

> lib/matplotlib/tests/test_image.py:1123
  /home/travis/build/matplotlib/matplotlib/lib/matplotlib/tests/test_image.py:1123: PytestUnknownMarkWarning: Unknown pytest.mark.network - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.network

Reply:
:facepalm: I was worried about that ran the test locally, missed that the warning was in output and then pushed.  Looking at that terminal again there is definitely a warning...